pub-critic/src/app/components/profileCard/ProfileCard.tsx

```diff
@@ -29,8 +32,24 @@ export const ProfileCard = ({
         <p>{likeScore} like score</p>
       </div>
       <div className={classes.follows}>
-        <p>{followers} followers</p>
-        <p>{following} following</p>
+        <Link
+          className={classes.link}
+          href={{
+            pathname: "/follows/" + id,
+            query: { tab: 1 },
+          }}
+        >
+          {followers} followers
+        </Link>
+        <Link
+          className={classes.link}
+          href={{
+            pathname: "/follows/" + id,
+            query: { tab: 0 },
+          }}
+        >
+          {following} following
```

Comment:
.

Reply:
while this does make sense and probably improves readability, I believe it would be too much text on the user card and would not look that compact, will keep this in mind but skipping it for now